app/models/event.rb

```diff
@@ -38,4 +40,14 @@ def exception_note(zip_code)
   def agency_name
     agency.loc_name
   end
+
+  def form_data
+    return '' unless forms[0]
+
+    # relies on forms data rule that the only difference
+    # in the forms per event is the language id, so just
+    # taking the first result
+    { display_age_adult: forms[0].max_age_child + 1,
+      display_age_senior: forms[0].max_age_adult + 1 }
```

Comment:
What's the purpose of `+ 1` here?

Reply:
@davidpickering requested this to match the client display.  For example the UI asks for number of adults(age 18+).  The database includes max_age_child which in this case is 17, so the display_age_adult is set to max_age_child + 1 (18).

Reply:
That seems reasonable. Could we move the `+ 1`s to a method on the form model?

Reply:
done.  I moved all of the form display to the form event model and serializer per Mahesh's suggestion.  See the description at the top of this PR to see what that looks like.